assets/components/contour/templates/03-envoy.yaml

```diff
@@ -142,7 +145,6 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-      # XXX: Lokomotive specific change
```

Comment:
This is not needed because upstream included `automountServiceAccountToken: false`.